src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -160,6 +169,21 @@ private void readFromNBT(NBTTagCompound compound)
             }
         }
 
+        // Restore colony achievements
+        final NBTTagList achievementTagList = compound.getTagList(TAG_ACHIEVEMENT_LIST, NBT.TAG_COMPOUND);
+        this.colonyAchievements = new ArrayList<>();
+        for (int i = 0; i < achievementTagList.tagCount(); ++i)
+        {
+            final NBTTagCompound achievementCompound = achievementTagList.getCompoundTagAt(i);
+            final String achievementKey = achievementCompound.getString(TAG_ACHIEVEMENT);
+            final StatBase statBase = StatList.getOneShotStat(achievementKey);
+            System.out.println("Checking stat: "+achievementKey+":"+statBase.statId);
+            if (statBase instanceof Achievement) {
+                System.out.println("Found an achievement");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS106)